docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,279 @@
+|bwc| HA Cluster - BETA
+=======================
+
+|bwc| (EWC) is the commercial version of the StackStorm automation platform. EWC adds priority
+support, advanced features such as fine-tuned access control, LDAP, and Workflow Designer. To
+learn more about |bwc|, get an evaluation license, or request a quote, visit `extremenetworks.com/product/workflow-composer
+<https://www.extremenetworks.com/product/workflow-composer/>`_.
+
+This document provides an installation blueprint for a Highly Availabile StackStorm Enterprise (|bwc|) cluster
+based on `Kubernetes <https://kubernetes.io/>`__, a container orchestration platform with a planet scale.
```

Comment:
"with a planet scale"? If you choose to use this term, perhaps replace with "at planet scale" instead? @LindsayHill thoughts?